pkgs/applications/audio/gradio/default.nix

```diff
@@ -13,13 +12,11 @@
 , vala
 , sqlite
 , gst_all_1
-, gst_plugins ? with gst_all_1; [ gst-plugins-good gst-plugins-ugly ]
 }:
-let
-  version = "7.2";
 
-in stdenv.mkDerivation rec {
+stdenv.mkDerivation rec {
   name = "gradio-${version}";
```

Comment:
You can use `pname = "gradio"` now, the name will be created automatically.

Reply:
Done.